### DISCUSSION TITLE
Favorites always starts empty

### DISCUSSION THREAD

**Discussion by sanderlv** [punxaphil/custom-sonos-card#197] [Q&A] [answered]
When clicking the star icon I always get an empty page named "all favorites". I have to click the right top arrow to show the list. How can I change that behavior?

**Comment by punxaphil** [comment]
https://support.sonos.com/en-ca/article/add-favorites-to-my-sonos

**Comment by sanderlv** [comment]
Not entirely what I meant... see what I mean:

![sonos card](https://github.com/johanfrick/custom-sonos-card/assets/16528916/1fa3175e-f9b1-40c6-a933-d6eda501e8bd)

**Reply by punxaphil** [reply]
Yes, you need to go into the official Sonos app and add favorites.

**Reply by sanderlv** [reply]
I did that but they do not show up in the custom sonos card in homeassitant. I do however see (as in the picture above) my manually added favorites in HA. But I only see them after one time after entering the card clicking on the icon right top. My question is how to avoid that right top icon clicking every time and directly show the HA favorites list.

**Comment by sanderlv** [comment]
Johan, not sure if you get my question correctly. I always need to press the right top icon before it even shows the favorites list. Can you make it so that that step is not needed? Thus when I click the star is shows the list directly?

And please, I opened the question,can you please open again, it is not resolved :-).

**Comment by punxaphil** [comment, marked as answer]
Fixed: https://github.com/johanfrick/custom-sonos-card/releases/tag/v6.11.3

**Reply by sanderlv** [reply]
Cool! Thanks!
